include/seccomp-syscalls.h

```diff
@@ -1097,6 +1118,12 @@
 
 #define __SNR_madvise			__NR_madvise
 
+#ifdef __NR_map_shadow_stack
+#define __SNR_map_shadow_stack		__NR_map_shadow_stack
+#else
+#define __SNR_map_shadow_stack		__PNR_map_shadow_stack
+#endif
+
```

Comment:
Isn't `map_shadow_stack()` defined for all the arches?

Reply:
Yes it is, but in [this commit](https://github.com/seccomp/libseccomp/commit/53267af3fb56eed93a50b8ef92f41825c97a7813) the PNR was allocated anyway and the change is already shipped in v2.5.5. So if we're to consider PNR values as part of the public API we would have to keep `__PNR_map_shadow_stack` defined, and for stylistic consistency we would have to keep this `#ifdef`-fery as well.

Reply:
Thanks for the reminder @xen0n.

In the manpages we "highly recommend" the use of the `SCMP_SYS()` macro as opposed to directly specifying syscall numbers so it's a bit of a gray area.  I think it would be okay for us to modify/remove the PNR syscall values if necessary, but we probably shouldn't do so unless we have a good reason.  Considering everything, it looks like you approach of leaving the PNR values as-is is likely the right one.  Thanks :)